healthcheck/core/src/main/java/org/apache/felix/hc/core/impl/executor/async/cron/CronScheduler.java

```diff
@@ -0,0 +1,170 @@
+package org.apache.felix.hc.core.impl.executor.async.cron;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+
+import java.time.Duration;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.RejectedExecutionException;
+import java.util.concurrent.TimeUnit;
+
+import org.slf4j.Logger;
```

Comment:
Is it intended to use SLF4j logger und not an OSGi logger?

Reply:
Since this class is not leveraging SCR, I used slf4j and slf4j is also used in many other places in Felix HC.